PowerEditor/installer/nativeLang/polish.xml

```diff
@@ -1258,41 +1258,39 @@ Kontynuować?"/> <!-- HowToReproduce: when you openned file is modified and save
             <SessionFileInvalidError title="Ładowanie sesji" message="Nie udało się załadować sesji. Plik sesji jest uszkodzony lub nieprawidłowy."/> <!-- HowToReproduce: Save current session via menu "File -> Save Session...", use another editor to modify the session you saved to make it an invalid xml file. Then load this invalid session via menu "File -> Load Session...". -->
-            <DroppingFolderAsProjectModeWarning title="Nieprawidłowe działanie" message="Pliki i foldery muszą być przenoszone do programu osobno, ponieważ aktywny jest tryb „Folder jako Projekt”.
-Żeby to zmienić, należy włączyć opcję „Nie traktuj przenoszonego folderu jako obszaru roboczego, tylko ładuj wszystkie jego pliki” w zakładce „Domyślna ścieżka” w oknie preferencji."/>
+            <DroppingFolderAsProjectModeWarning title="Nieprawidłowe działanie" message="Pliki i foldery muszą być przenoszone do programu osobno, ponieważ aktywny jest tryb obszaru roboczego.
+Żeby to zmienić, należy włączyć opcję „Nie traktuj przenoszonego folderu jako obszaru roboczego, tylko ładuj wszystkie jego pliki” w zakładce „Domyślna ścieżka” w oknie ustawień."/>
             <SortingError title="Błąd sortowania" message="Nie można wykonać sortowania przez linię numer $INT_REPLACE$."/> <!-- HowToReproduce: this message prevents from system failure. It's hard to reproduce. -->
-            <ColumnModeTip title="Wskazówka dotycząca edycji kolumnowej" message="ALT+LPM oraz Alt+Shift+Strzałka przełączają tryb kolumnowy."/>
+            <ColumnModeTip title="Wskazówka dotycząca edycji kolumnowej" message="Alt+LPM oraz Alt+Shift+Strzałka przełączają tryb kolumnowy."/>
             <BufferInvalidWarning title="Błąd zapisywania" message="Nie udało się zapisać. Błąd bufora."/> <!-- HowToReproduce: this message prevents from system failure. It's hard to reproduce. -->
             <DoCloseOrNot title="Zachowywanie nieistniejącego pliku" message="Plik „$STR_REPLACE$” nie istnieje.
 Czy mimo to zachować go w edytorze?"/>
             <DoDeleteOrNot title="Usuwanie" message="Plik „$STR_REPLACE$”
-zostanie zamknięty i przeniesiony do Kosza.
+zostanie zamknięty i przeniesiony do kosza.
 Kontynuować?"/>
             <NoBackupDoSaveFile title="Zapisywanie" message="Nie udało się odnaleźć kopii zapasowej na dysku. Została usunięta.
 Żeby nie stracić danych, należy zapisać bieżące zmiany.
 Zapisać plik „$STR_REPLACE$”?"/>
-            <DoReloadOrNot title="Przeładowywanie" message="Plik „&quot;$STR_REPLACE$”
+            <DoReloadOrNot title="Przeładowywanie" message="Plik „$STR_REPLACE$” został zmodyfikowany przez inny program.
 
-został zmodyfikowany przez inny program.
 Przeładować go?"/>
-            <DoReloadOrNotAndLooseChange title="Przeładowywanie" message="Plik „$STR_REPLACE$”
+            <DoReloadOrNotAndLooseChange title="Przeładowywanie" message="Plik „$STR_REPLACE$” został zmodyfikowany przez inny program.
 
-został zmodyfikowany przez inny program.
```

Comment:
@przemo20 Please don't change the number of lines relative to the original english file. Revert to the previous variant or use another sentence (but keep the number of lines).